runbot_travis2docker/models/runbot_build.py

```diff
@@ -144,6 +144,10 @@ def _checkout(self):
                 branch_short_name, '--root-path=' + t2d_path,
                 '--exclude-after-success',
                 '--docker-image=%s' % build.repo_id.travis2docker_image,
+                # Avoid corruption of postgresql
+                '--runs-at-the-end-script=pg_isready -q && '
+                '/etc/init.d/postgresql stop && '
+                'rm -f /var/run/postgresql/.s.PGSQL.5432*',
```

Comment:
Why do you need to remove this file? Shouldn't it be automatically removed if everything goes OK when stopping the PostgreSQL service?

In case this is really needed, I'd like to suggest adding the `-v` parameter, to be able to log and know when this file was manually removed:  
`rm -fv /var/run/postgresql/.s.PGSQL.5432*`

Reply:
This is a recommendation of @gurneyalex from 
https://github.com/Vauxoo/travis2docker/pull/119

I think it is uneeded but he tested this part and it is working.
Using verbose mode lets to see in the log what cases is removing it manually

Reply:
@luisg123v 
The new build is not showing the deleting of the file.
Then I will remove this part of the code.

https://travis-ci.org/OCA/runbot-addons/jobs/426694697#L1594